static/src/js/components/OneOfField/OneOfField.jsx

```diff
@@ -98,12 +101,9 @@ class OneOfField extends React.Component {
       }
     }
 
-    /*
-    *  The code below commented out caused a bug where it would revert the option selected in some cases.
-    */
-    // if (newState !== this.state) {
-    //   this.setState(newState)
-    // }
+    if (newState !== this.state) {
```

Comment:
Why are these changes showing up in your PR?

Reply:
rebase issue. Deep fixed.